community/libgit2/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+cmake -B build \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_INSTALL_LIBDIR=/usr/lib \
+    -DBUILD_SHARED_LIBS=OFF
```

Comment:
Why disable shared libraries? Nothing needs libgit2 yet so not too bothered about this right now.

Reply:
Yeah, to my knowledge there's nothing that depends on `libgit2` implicitly or otherwise so shared libs didn't really come off as necessary.

Personally I only needed a few header files from the package 😅